requirements.txt

```diff
@@ -1,10 +1,10 @@
 Django==2.2
 django-prometheus==1.0.15
-canonicalwebteam.blog==1.5.0
+git+https://github.com/b-m-f/blog-extension@group-pages#egg=canonicalwebteam.blog
```

Comment:
Is there now a new version that this could be pointed at?

Reply:
Have updated it to `1.5.1`. Thank you.